cuda_bindings/setup.py

```diff
@@ -329,7 +329,7 @@ def do_cythonize(extensions):
     return cythonize(
         extensions,
         nthreads=nthreads,
-        compiler_directives=dict(language_level=3, embedsignature=True, binding=True),
+        compiler_directives=dict(language_level=3, embedsignature=True, binding=True, freethreading_compatible=True),
```

Comment:
FWIW Cython's doc is really unclear on this. I hope this flag does nothing for regular builds...